app/exposureHistory.ts

```diff
@@ -0,0 +1,37 @@
+import dayjs from 'dayjs';
+
+export type Posix = number;
+
+export interface Possible {
```

Comment:
idk if the GPS side will have the same data requirements for an `ExposureDatum` but if not we could introduce a generic over the history to account for this.  something like:
```
export type ExposureHistory<T> =T[];
```

Reply:
I think it will need to end up being a little different, but that's not well defined right now. From what I see, that would need only a corresponding change at the exposureHistory details component level?

Reply:
yup totally.  which given that that component is a leaf node i dont see any risks there.